### PR TITLE
Add new RP account flag

### LINKED ISSUE #2048: add "this is a RP account" settings flag
https://dw-suggestions.dreamwidth.org/1494165.html

Add a way for people to flag their accounts as RP accounts, that will display on the profile

### PR DESCRIPTION
**Pull request by rahaeli**
Fixes #2048.

Adds a new "rp account" setting; toggling it prints a blurb on the profile
saying that the account is an RP account, and in the future may control
access to features that only make sense for RP accounts.

### CODE REVIEW

File: htdocs/profile.bml [hunk 1]
```diff
@@ -396,7 +396,11 @@ body<=
                           style => 'vertical-align: middle; border: 0;' } );
         $ret .= "</a>\n";
     }
+
+    if ( $u->is_rp_account ) {
+        $ret .= ": " . BML::ml( '.details.rp' );
     $ret .= q{ </div> };
```
Comment (zorkian) [inline htdocs/profile.bml]: I think this line has to move to after the next line -- we want to keep the `</div>` whether or not the user is flagged as an RP account.

Comment (rahaeli) [inline htdocs/profile.bml]: Oops! Fixed

**Comment by zorkian** [comment]
AMAZING GOLD STAR 🗡

**Comment by zorkian** [comment]
TIL that colon-D autocompletes to 🗡 ...

**Comment by kareila** [comment]
🌟

**Comment by azurelunatic** [comment]
Is this a feature that would make sense reflecting in the stats?

**Comment by rahaeli** [comment]
Possibly in the future, yes!

**Pull request merged**